### DISCUSSION TITLE
First boot of the Fly-by-wire A320. Feedback: Two small show stoppers...

### DISCUSSION THREAD

**Discussion by mikkelen** [flybywiresim/aircraft#6587] [General]
Looking forward to testing this aircraft more in the future, however I would just like to give feedback on the two first issues with the aircraft I encountered:

- The reverse thrust sound seems unrealistically loud, it also seemed to be of a worse recording quality than the default sounds
- I bought the Thrustmaster Airbus Captain pack, and the throttles does not work out of the box

**Comment by CptBayer** [comment]
Hi,

calibrate it via the EFB ..... 

greets  ...

**Comment by mikkelen** [comment]
As it works out of the box with the default aircraft, maybe this is something that can be looked into? Where would you like to have the requests logged? (I'm not used to Github)

**Reply by tracernz** [reply]
The way it "works" out of the box leaves the throttle axis between idle and CL detent extremely non-linear/unusable. You need to calibrate the positions of the detents to get proper behaviour.

**Reply by mikkelen** [reply]
ok - I've only flown the landing tutorial with the default AC and did not notice. 

I'm feeling more enclined to test the aircraft, it's good to know why something works in another way than expected :) 

but do you know what's the issue with the reverser sound?
